src/clr-icons/shapes/essential-shapes.ts

```diff
@@ -942,6 +928,9 @@ export const EssentialShapes: any = {
   wand: ClrShapeWand,
 };
 
+// Aliased, and reexported
```

Comment:
we haven't reassigned the shapes to different names so saying *aliased* in the comment might be a bit off. I think it's just re-exporting the shapes from `core-shapes`. But aside from that, looks good.